src/benchmark_mpi.in.h

```diff
@@ -0,0 +1,55 @@
+#ifndef BENCHMARK_MPI_H
+#define BENCHMARK_MPI_H
+
+#cmakedefine BENCHMARK_ENABLE_MPI
```

Comment:
This file in general and this line in particular are making it impossible to compile the project without using cmake.  That's not acceptable: the code should be self-standing and compilable with any modern C++ compiler, at least in the case where mpi is not being used.